app/dashboard/models.py

```diff
@@ -244,6 +244,7 @@ class Bounty(SuperModel):
     idx_project_length = models.IntegerField(default=0, db_index=True)
     idx_status = models.CharField(max_length=9, choices=STATUS_CHOICES, default='open', db_index=True)
     issue_description = models.TextField(default='', blank=True)
+    funding_organisation = models.TextField(null=True, blank=True)
```

Comment:
@SaptakS Will this ever be greater than 255 characters? If not, would you mind switching it to a `models.CharField(blank=True, max_length=255)` or whatever the max length might be (probably less...  100 maybe?) and dropping the `null=True` - Typically, you don't want to set `null=True` on `TextField` or `CharField`.

Following the change, can you delete and recreate the migration?

Reply:
Okay. I will put `default=''` in that case